code/__DEFINES/contracts.dm

```diff
@@ -0,0 +1,43 @@
+#define CONTRACT_POWER "power"
+#define CONTRACT_WEALTH "wealth"
+#define CONTRACT_PRESTIGE "prestige"
+#define CONTRACT_MAGIC "magic"
+#define CONTRACT_REVIVE "revive"
+#define CONTRACT_KNOWLEDGE "knowledge"
+#define CONTRACT_UNWILLING "unwilling"
+
+#define BANE_SALT "salt"
+#define BANE_LIGHT "light"
+#define BANE_IRON "iron"
+#define BANE_WHITECLOTHES "whiteclothes"
+#define BANE_SILVER "silver"
+#define BANE_HARVEST "harvest"
+#define BANE_TOOLBOX "toolbox"
+
+#define OBLIGATION_FOOD "food"
+#define OBLIGATION_FIDDLE "fiddle"
+#define OBLIGATION_DANCEOFF "danceoff"
+#define OBLIGATION_GREET "greet"
+#define OBLIGATION_PRESENCEKNOWN "presenceknown"
+#define OBLIGATION_SAYNAME "sayname"
+#define OBLIGATION_ANNOUNCEKILL "announcekill"
+#define OBLIGATION_ANSWERTONAME "answername"
+
+#define BAN_HURTWOMAN "hurtwoman"
+#define BAN_CHAPEL "chapel"
+#define BAN_HURTPRIEST "hurtpriest"
+#define BAN_AVOIDWATER "avoidwater"
+#define BAN_STRIKEUNCONCIOUS "strikeunconcious"
+#define BAN_HURTLIZARD "hurtlizard"
```

Comment:
non balance related point: should we add in other races since lizards i kinda a TG thing and pick at random from them?